lib/tasks/import_pricing_maps.rake

```diff
@@ -66,15 +69,16 @@ namespace :data do
       puts "#"*50
       puts "Starting import"
       input_file = Rails.root.join("db", "imports", file)
-      CSV.foreach(input_file, :headers => true) do |row|
-        service = Service.find(row['service_id'].to_i)
+      CSV.foreach(input_file, :headers => true, skip_blanks: true, skip_lines: /^(?:,\s*)+$/) do |row|
+
+        service = Service.find(row['Service ID'].to_i)
 
         pricing_map = service.pricing_maps.build(
                                               :full_rate => Service.dollars_to_cents(row['full_rate'].to_s.strip.gsub("$", "").gsub(",", "")),
-                                              :corporate_rate => (row['corporate_rate'].blank? ? nil : Service.dollars_to_cents(row['corporate_rate'].to_s.strip.gsub("$", "").gsub(",", ""))),
-                                              :federal_rate => (row['federal_rate'].blank? ? nil : Service.dollars_to_cents(row['federal_rate'].to_s.strip.gsub("$", "").gsub(",", ""))),
-                                              :member_rate => (row['member_rate'].blank? ? nil : Service.dollars_to_cents(row['member_rate'].to_s.strip.gsub("$", "").gsub(",", ""))),
-                                              :other_rate => (row['other_rate'].blank? ? nil : Service.dollars_to_cents(row['other_rate'].to_s.strip.gsub("$", "").gsub(",", ""))),
+                                              :corporate_rate => (row['corporate_rate'].nil? || row['corporate_rate'].blank? ? nil : Service.dollars_to_cents(row['corporate_rate'].to_s.strip.gsub("$", "").gsub(",", ""))),
```

Comment:
Doesn't blank? check for "" and nil.  See https://apidock.com/rails/Object/blank%3f